SniperService.cs

```diff
@@ -192,6 +192,7 @@ private async Task PairCreated(PairCreatedEvent pair)
                     Log.Logger.Information("Buying Token pair: {0} WHITELISTED ADDRESS: {1}", symbol, addressWhitelisted);
                 }
                 await _tradeHandler.Buy(otherPairAddress, otherTokenIdx, pair.Pair, _sniperConfig.AmountToSnipe);
+                await _tradeHandler.Approve(otherPairAddress);
```

Comment:
Is there a reason we approve the pair after buying?

Reply:
Yes. Assuming you turn of the honeypot detection, It never approves the pair before the actual sell. Therefore the sell will fail.